android/engine/src/main/java/org/smartregister/fhircore/engine/datastore/ProtoDataStore.kt

```diff
@@ -20,30 +20,30 @@ import android.content.Context
 import androidx.datastore.core.DataStore
 import androidx.datastore.dataStore
 import dagger.hilt.android.qualifiers.ApplicationContext
-import java.io.IOException
-import javax.inject.Inject
-import javax.inject.Singleton
 import kotlinx.coroutines.flow.catch
-import org.smartregister.fhircore.engine.datastore.mockdata.PractitionerDetails
-import org.smartregister.fhircore.engine.datastore.mockdata.UserInfo
+import org.smartregister.fhircore.engine.data.remote.model.response.PractitionerDetails
+import org.smartregister.fhircore.engine.data.remote.model.response.UserInfo
 import org.smartregister.fhircore.engine.datastore.serializers.PractitionerDetailsDataStoreSerializer
 import org.smartregister.fhircore.engine.datastore.serializers.UserInfoDataStoreSerializer
 import timber.log.Timber
+import java.io.IOException
+import javax.inject.Inject
+import javax.inject.Singleton
 
 private const val PRACTITIONER_DETAILS_DATASTORE_JSON = "practitioner_details.json"
 private const val USER_INFO_DATASTORE_JSON = "user_info.json"
 private const val TAG = "Proto DataStore"
 
 val Context.practitionerProtoStore: DataStore<PractitionerDetails> by
   dataStore(
-    fileName = PRACTITIONER_DETAILS_DATASTORE_JSON,
-    serializer = PractitionerDetailsDataStoreSerializer,
+          fileName = PRACTITIONER_DETAILS_DATASTORE_JSON,
+          serializer = PractitionerDetailsDataStoreSerializer,
   )
 
 val Context.userInfoProtoStore: DataStore<UserInfo> by
   dataStore(
-    fileName = USER_INFO_DATASTORE_JSON,
-    serializer = UserInfoDataStoreSerializer,
+          fileName = USER_INFO_DATASTORE_JSON,
+          serializer = UserInfoDataStoreSerializer,
```

Comment:
Looks like extra leading spaces were applied. Did you run `spotlessApply`?

Reply:
No, PR is not ready yet. Would you advise to do spotless on every commit?

Reply:
Okay. Running spotlessApply after every commit would be preferred. You can run it once when the PR is ready.

Reply:
Enable `ktfmt` to format the code directly in the IDE. Follow the setup here in the [Developer setup docs](https://docs.opensrp.io/engineering/android-app/developer-setup)